dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.messaging-3.0.internal.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.messaging-3.0.internal
+singleton=true
+IBM-App-ForceRestart: uninstall
+IBM-API-Package: jakarta.jms; version="3.0"; type="spec"
+-features=io.openliberty.jakarta.messaging-3.0, \
+ io.openliberty.jakarta.connectors-2.0, \
+ io.openliberty.connectors-2.0.internal, \
```

Comment:
`io.openliberty.jakarta.connectors-2.0` should be the only feature you need here as it pulls in `io.openliberty.connectors-2.0.internal`

Reply:
Looking at the feature descriptors, `io.openliberty.jakarta.connectors-2.0` provisions the API bundle, only.